js/render-and-events.js

```diff
@@ -1,4 +1,30 @@
 import {getPhotosSpecificationList} from './data.js';
-import {renderPicture, renderPictures} from './picture-block.js';
+import {renderPicture, renderPictures, photosPool} from './picture-block.js';
+import {renderBigPhotosPopup, bigPhotosPopup} from './big-picture-block.js';
 
-renderPictures(getPhotosSpecificationList());
+const closePhotosPopupElement = document.querySelector('.big-picture__cancel');
+
+const photosSpecificationList = getPhotosSpecificationList();
+
+const bigPhotosPopupConditionElement = document.querySelector('body');
+
+renderPictures(photosSpecificationList);
+
+const openBigPhotosPopup = (evt) => {
+  bigPhotosPopupConditionElement.classList.add('modal-open');
+  bigPhotosPopup.classList.remove('hidden');
+  removePicturesEventListeners(evt.target);
+  renderBigPhotosPopup(evt.target);
+  closePhotosPopupElement.addEventListener('click', closeBigPhotosPopup);
+};
```

Comment:
Тут обработчик превращается в замыкание, которое принимает данные и возвращает сам обработчик, в области видимости которого будут доступны данные из  `pictureData `.
```suggestion
const openBigPhotosPopup = (pictureData) => (event) => {
  bigPhotosPopupConditionElement.classList.add('modal-open');
  bigPhotosPopup.classList.remove('hidden');
  removePicturesEventListeners(evt.target);
  renderBigPhotosPopup(evt.target);
  closePhotosPopupElement.addEventListener('click', closeBigPhotosPopup);
};
```